cmd/influx_stress/example.toml

```diff
@@ -0,0 +1,82 @@
+[write]
+  concurrency = 10
+  batch_size = 5000
+  batch_interval = "0s"
+  database = "stress"
+  precision = "n"
+  address = "localhost:8086"
+  reset_database = true
+  # doesnt do anything
```

Comment:
the `starting_point` config option currently doesn't do anything. Eventually the idea is that it will be used to specify the timestamp for the first point that gets written during the test.